.github/workflows/metadata-update.yml

```diff
@@ -0,0 +1,85 @@
+name: Metadata Update
+
+on:
+  workflow_dispatch:  # allow this to be manually triggered
+  schedule:
+    - cron: "00 1 * * *" # daily at 1:00 UTC
+
+permissions:
+  contents: read
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # for git push to PR branch
+      pull-requests: write # for creating PRs
+
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Free disk space
+        run: .github/scripts/gha-free-disk-space.sh
+
+      - name: Set up JDK for running Gradle
+        uses: actions/setup-java@c5195efecf7bdfc987ee8bae7a71cb8b11521c00 # v4.7.1
+        with:
+          distribution: temurin
+          java-version-file: .java-version
+
+      - name: Set up gradle
+        uses: gradle/actions/setup-gradle@ac638b010cf58a27ee6c972d7336334ccaf61c96 # v4.4.1
+
+      - name: Collect telemetry
+        run: ./instrumentation-docs/ci-collect.sh
+
+      - name: Run documentation analyzer
+        run: ./gradlew :instrumentation-docs:runAnalysis
+
+      - name: Check for diff
```

Comment:
should we also check somewhere to see if there's already a PR open? i.e. in case we don't merge it right away, will the current workflow create duplicate PRs each day?

Reply:
good catch! updated